install-latest.sh

```diff
@@ -341,6 +343,29 @@ End of functions from https://github.com/client9/shlib
 ------------------------------------------------------------------------
 EOF
 
+###############################################
+# Gets the binary filename (without extentsion)
+# used in repo release.
+#
+# Globals:
+#   PROJECT_NAME
+#   VERSION
+#   OS
+#   ARCH
+# Arguments:
+#   None
+################################################
+get_binary_name() {
+  name=${PROJECT_NAME}_${VERSION}_${OS}_${ARCH}
+  case ${PLATFORM} in
+    darwin/arm64)
+      log_info "Platform ${PLATFORM} (m1 silicon) detected, using compatible darwin/amd64 binary instead."
+      name=${PROJECT_NAME}_${VERSION}_${OS}_${ARCH}
```

Comment:
Looks like this case statement only logs that M1 arch is found, but doesn't change the binary name.

Reply:
Copy/pasted stale code from my m1 home machine - fixed now.